.editorconfig

```diff
@@ -0,0 +1,32 @@
+# EditorConfig: http://EditorConfig.org
+
+root = true
+
+[*]
+indent_style = tab
+indent_size = 2
+insert_final_newline = true
+trim_trailing_whitespace = true
+end_of_line = lf
+charset = utf-8
+max_line_length = 99
+
+[{*.html,*.css,*.js}]
+max_line_length = off
+
+[*.py]
+indent_size = 4
+indent_style = space
```

Comment:
We may as well remove the line-length setting in our pyproject.toml and move it here.

I added the line length in the previous PR to match Hatch's defaults.

Reply:
I think it needs to remain in both locations, or else ruff won't be able to enforce it.  i'm not sure that editorconfig is strict enough.  I will sync it though.